apps/test/unit/gamelab/GameLabSpriteTest.js

```diff
@@ -150,7 +150,7 @@ describe('GameLabSprite', function () {
       expect(isTouching5to6).to.equal(true).and.to.equal(isTouching5to6);
     });
 
-    it('does not affect the location or velocity of the sprite', function () {
+    it('does not affect the location of the sprite', function () {
```

Comment:
This test's name said it checked that velocity was not modified, but the test itself only checked positions :grin: which explains how this slipped through the cracks before.